### PR TITLE
feat: connect to cloud use url and token

### PR DESCRIPTION
**Pull request by dingbo8128**

### CODE REVIEW

**Review by sangshuduo — APPROVED** [review]
LGTM

File: src/main/java/com/taosdata/jdbc/rs/RestfulConnection.java [hunk 1]
```diff
@@ -36,6 +38,42 @@ public RestfulConnection(String host, String port, Properties props, String data
...
+        super(new Properties());
+        clientInfoProps.setProperty(TSDBDriver.PROPERTY_KEY_TIMESTAMP_FORMAT, String.valueOf(TimestampFormat.TIMESTAMP));
+
+        int poolSize = Integer.parseInt(HttpClientPoolUtil.DEFAULT_MAX_PER_ROUTE);
+        HttpClientPoolUtil.init(poolSize, true);
+        if (url == null || token == null) {
+            throw new Exception("url and token can't be null");
+        }
+        if (!url.startsWith("http://") && !url.startsWith("https://")) {
+            System.out.println("No protocol specified in url, use \"http://\" by default");
```
Comment (huolibo) [inline src/main/java/com/taosdata/jdbc/rs/RestfulConnection.java:57]: I do not  think System.out is suitable for this connector. you can just set http or add logback or log2j

File: src/main/java/com/taosdata/jdbc/rs/RestfulConnection.java [hunk 2]
```diff
@@ -36,6 +38,42 @@ public RestfulConnection(String host, String port, Properties props, String data
...
+        }
+
+        this.url = url;
+        this.useSsl = url.startsWith("https://");
+        String tmpSchema = this.useSsl ? "https://" : "http://";
+        String hostPort = url.replaceFirst(tmpSchema, "");
+        String[] splits = hostPort.split(":");
```
Comment (huolibo) [inline src/main/java/com/taosdata/jdbc/rs/RestfulConnection.java:64]: you can use 'com.taosdata.jdbc.AbstractDriver#parseURL' to parse url.

**Review by huolibo — COMMENTED** [review]
See comments

File: src/main/java/com/taosdata/jdbc/rs/RestfulConnection.java (hunk 2 above)
Comment (dingbo8128) [reply src/main/java/com/taosdata/jdbc/rs/RestfulConnection.java:60]: It's too heavy for this simple processing

**Review by huolibo — APPROVED** [review]

**Pull request merged**